packages/sdk-js/src/feature-repository.ts

```diff
@@ -458,24 +500,21 @@ function disableChannel(channel: ScopedChannel) {
   channel.src.onerror = null;
   channel.src.close();
   channel.src = null;
+  if (channel.state === "active") {
+    channel.state = "disabled";
+  }
```

Comment:
So what happens if a channel is disabled while it's in an idle state?   Wouldn't it then switch back to active when the page is focused?

Reply:
This (probably) can't happen: we control when a channel gets disabled or idled, and we don't disable anything but active channels. I will add another safeguard just to be sure though...

The reverse can happen though: a disabled channel can be converted to idle, which should be fine given that 99% of the time it's disabled because of some connection error with exponential backoff, and waking up from idle state is a good enough alternative to backoff/retry.